kalite/topic_tools/annotate.py

```diff
@@ -61,8 +61,7 @@ def update_content_availability(content_list, language="en", channel="khan"):
 
             # Databases have been pre-filtered to only contain existing exercises
             # Assume if the assessment items have been downloaded, then everything is hunky dory.
-            if os.path.exists(contentload_settings.KHAN_ASSESSMENT_ITEM_VERSION_PATH):
-                update["available"] = True
+            update["available"] = True
```

Comment:
Turns out we also have to mark topic availability on the `content-pack-maker` side as well.